fixtures/u2_func_simple.rb.sem.uast

```diff
@@ -0,0 +1,166 @@
+                                 end: { '@type': "uast:Position",
+                                    offset: 40,
+                                    line: 3,
+                                    col: 12,
+                                 },
+                              },
+                              Name: "a",
+                           },
+                        },
+                     ],
+                  },
+               ],
+               Type: { '@type': "uast:FunctionType",
+                  Arguments: { '@type': "args",
+                     '@role': [Argument, Expression, Incomplete],
+                     '@pos': { '@type': "uast:Positions",
+                        start: { '@type': "uast:Position",
+                           offset: 12,
+                           line: 1,
+                           col: 13,
+                        },
+                        end: { '@type': "uast:Position",
+                           offset: 17,
+                           line: 1,
+                           col: 18,
+                        },
+                     },
+                     children: [
+                        { '@type': "uast:Argument",
+                           '@pos': { '@type': "uast:Positions",
+                              start: { '@type': "uast:Position",
+                                 offset: 13,
+                                 line: 1,
+                                 col: 14,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 13,
+                                 line: 1,
+                                 col: 14,
+                              },
+                           },
+                           Init: ~,
+                           MapVariadic: false,
+                           Name: "a",
+                           Receiver: false,
+                           Type: ~,
+                           Variadic: false,
+                        },
+                        { '@type': "uast:Argument",
+                           '@pos': { '@type': "uast:Positions",
+                              start: { '@type': "uast:Position",
+                                 offset: 16,
+                                 line: 1,
+                                 col: 17,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 16,
+                                 line: 1,
+                                 col: 17,
+                              },
+                           },
+                           Init: ~,
+                           MapVariadic: false,
+                           Name: "b",
+                           Receiver: false,
+                           Type: ~,
+                           Variadic: false,
+                        },
+                     ],
+                  },
+                  Returns: ~,
```

Comment:
Ruby always has exactly one return with unknown type (defaults to `nil` or the last dropped expression)

Reply:
We would need to actually parse the code to know that since it's not in the native AST so forwarding what it gives us is the only option for now.